ssl/t1_lib.c

```diff
@@ -1196,13 +1410,35 @@ int ssl_setup_sig_algs(SSL_CTX *ctx)
             cache[i].enabled = 0;
         EVP_PKEY_CTX_free(pctx);
     }
+
+    /* Now complete cache and tls12_sigalgs list with provider sig information */
+    cache_idx = OSSL_NELEM(sigalg_lookup_tbl);
+    for (i=0; i<ctx->sigalg_list_len; i++) {
```

Comment:
nit: spaces around `=` and `<`

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.